src/data/polynomial/monic.lean

```diff
@@ -334,6 +334,9 @@ begin
   rw [← leading_coeff_of_injective hf, hp.leading_coeff, f.map_one]
 end
 
+theorem monic_map_iff_of_injective [nontrivial S] {p : R[X]} : p.monic ↔ (p.map f).monic :=
```

Comment:
Oh I see, you inadvertently introduced a `polynomial.function` namespace by adding this lemma here. I think the name you want is `_root_.function.injective.monic_map_iff`. 

You need to remove `nontrivial` since it's not used and the linter will complain.
```suggestion
theorem monic_map_iff_of_injective {p : R[X]} : p.monic ↔ (p.map f).monic :=
```

Reply:
Ok great thanks! Just made the change

Reply:
Maybe you didn't notice, but the linter has [found](https://github.com/leanprover-community/mathlib/actions/runs/3864036073/jobs/6586910025) four more declarations with unused arguments.